doc/OnlineDocs/contribution_guide.rst

```diff
@@ -75,6 +75,35 @@ at least 70% coverage of the lines modified in the PR and prefer coverage
 closer to 90%. We also require that all tests pass before a PR will be 
 merged.
 
+Tests must import the Pyomo test harness from
+``pyomo.common.unittest`` instead of using Python's built-in
+``unittest`` module directly. This wrapper extends the standard testing
+framework with Pyomo-specific capabilities, including automatic plugin
+discovery, solver availability checks, and improved test filtering
+controls. Using the provided interface ensures that all tests run
```

Comment:
This is not technically true (the availability / plugin / filtering from `conftest.py` has nothing to do with Pyomo's custom unittest module.
```suggestion
framework with Pyomo-specific capabilities, including support for test
timeouts and Pyomo-specific assertions for comparing expressions and
nested containers with numerical tolerance.
Using the provided interface ensures that all tests run
```